tests/system.py

```diff
@@ -1055,8 +1055,9 @@ def test_create_signed_read_url_v2_w_access_token(self):
         client = iam_credentials_v1.IAMCredentialsClient()
         service_account_email = Config.CLIENT._credentials.service_account_email
         name = client.service_account_path("-", service_account_email)
-        scope = ["https://www.googleapis.com/auth/devstorage.read_write"]
+        scope = ["https://www.googleapis.com/auth/cloud-platform"]
         response = client.generate_access_token(name, scope)
+        print(response.access_token)
```

Comment:
Is this leftover from debugging by chance?

Reply:
It should definitely go:  we don't want stray output in normal test runs.